src/ufw.yml

```
@@ -5,37 +5,47 @@
   become_method: sudo
   roles:
     - ufw
-  vars:
-    # Do not actually enforce firewall rules.  The assessors will turn
-    # UFW on if they want it.
-    state: disabled
 
 - hosts: all
   name: Basic configuration for UFW
   become: yes
   become_method: sudo
   tasks:
-    - name: Allow incoming traffic through default TCP ports
+    - name: Allow incoming traffic through specified TCP ports
       community.general.ufw:
         proto: tcp
         rule: allow
         to_port: "{{ item }}"
       loop:
         - "22"       # ssh for Guacamole server (internal to assessment env)
         - "25"       # Mail (external)
-        - "53"       # C2 traffic (external)
-        - "80"       # Domain verification (external)
-        - "443"      # Payload hosting (external)
         - "587"      # Mail (external)
         - "993"      # IMAP (internal to assessment env)
+        - 5000:5999  # Additional listeners, if necessary (external)
         - "5901"     # vnc for Guacamole (internal to assessment env)
-        - "8080"     # C2 listening and testing (external)
-        - 8000:8999  # Additional listeners, if necessary (external)
         - "50050"    # CobaltStrike (internal to assessment env)
-    - name: Allow incoming traffic through default UDP ports
+    - name: Deny incoming traffic to specified TCP ports
+      # These rules may be manually modified by assessors as needed, but we
+      # want to default to a minimal set of allowed ports.
+      #
+      # Note that these "deny" rules were explicitly requested in
+      # cisagov/cool-system-internal#80.  For additional context, see:
+      # https://github.com/cisagov/kali-packer/pull/92#discussion_r796234923
+      community.general.ufw:
+        proto: tcp
+        rule: deny
+        to_port: "{{ item }}"
+      loop:
+        - "53"       # C2 traffic (external)
+        - "80"       # Domain verification (external)
+        - "443"      # Payload hosting (external)
+        - "8080"     # C2 listening and testing (external)
```

Since this is a `deny` list I don't think the `(external)` markers make sense. I also thought the entry descriptions should mirror the linked issue. 

```suggestion
        - "53"       # DNS
        - "80"       # Domain verification
        - "443"      # Payload hosting
        - "8080"     # C2 listening
```